src/fu-security-attr.c

```diff
@@ -207,6 +207,314 @@ fu_security_attr_get_name(FwupdSecurityAttr *attr)
 	return g_strdup(fwupd_security_attr_get_name(attr));
 }
 
+const gchar *
+fu_security_attr_get_title(FwupdSecurityAttr *attr)
+{
+	const gchar *appstream_id = fwupd_security_attr_get_appstream_id(attr);
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SPI_BIOSWE) == 0) {
+		/* TRANSLATORS: Title: SPI refers to the flash chip in the computer */
+		return _("SPI Write");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SPI_BLE) == 0) {
+		/* TRANSLATORS: Title: SPI refers to the flash chip in the computer */
+		return _("SPI Lock");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SPI_SMM_BWP) == 0) {
+		/* TRANSLATORS: Title: SPI refers to the flash chip in the computer */
+		return _("SPI BIOS Region");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SPI_DESCRIPTOR) == 0) {
+		/* TRANSLATORS: Title: SPI refers to the flash chip in the computer */
+		return _("SPI BIOS Descriptor");
```

Comment:
All of these I know SPI is the correct term but I think you should go for "Flash Chip" because that's a lot more accessible of a description.

Reply:
What about just `Firmware`? Most users won't know what flash is either.

Reply:
Great idea

Reply:
I've made a lot of these nicer to read.